src/core/api/disputes/disputes.api.ts

```diff
@@ -0,0 +1,16 @@
+import { get, post } from 'src/core/api/http';
+
+import { DisputeReq, DisputeRes, DisputesPaginateRes, RespondDisputeReq } from './disputes.types';
+import { FilterReq } from '../types';
+
+export async function issueDispute(payload: DisputeReq): Promise<DisputeRes> {
+  return (await post<DisputeRes>('disputes', payload)).data;
+}
+
+export async function respondDispute(payload: RespondDisputeReq, disputedId): Promise<any> {
+  return (await post<any>(`disputes/${disputedId}/response`, payload)).data;
```

Comment:
This pull request is linked to another pull request. I didn't know about the outcome API because it requires details from the page you are working on. For now, I had to put 'any' but I won't merge this pull request. I just want you to review it, so let's skip it for now.
as you see in overview section, there is non checked input (first should merge https://github.com/socious-io/web-app-v2/pull/677)